packages/components/src/DateTimePickers/pickers/IncrementableScrollActionList/IncrementableScrollList/IncrementableScrollList.component.js

```diff
@@ -35,7 +47,17 @@ class IncrementableScrollList extends React.Component {
 	}
 
 	scrollRows(increment) {
-		const newRowIndex = this.state.startIndex + increment;
+		const firstIndex = 0;
+		const lastIndex = this.props.items.length - NB_ITEMS_DISPLAYED;
+		const needToScroll = increment < 0 && this.state.startIndex > firstIndex ||
```

Comment:
The 2 comments were tempting at first, but after a few headache, there is some problems :(.

Here : in the case we are positioned on firstIndex, we can't scroll before, ok, but we can't scroll after neither, not ok (We will have infinite sticky border, min max).

In the next comment : This is not always equal, in the case of positioning on index 3 (min to index 0). we ask for the 5 before so index -2 (3-5). But in that case we want don't want to go to an inexistant index, ok, but we still want to go to 0 and not stay where we at, not ok. Because we cannot see index 0, 1 and 2. It's why the keepInBoundaries.

Reply:
Oh you're right, I misread the method.